paymentsheet/src/main/java/com/stripe/android/paymentsheet/address/AddressFieldElementRepository.kt

```diff
@@ -4,15 +4,18 @@ import android.content.res.Resources
 import androidx.annotation.VisibleForTesting
 import com.stripe.android.paymentsheet.SectionFieldElement
 import com.stripe.android.paymentsheet.forms.transform
+import javax.inject.Inject
 
-internal object AddressFieldElementRepository {
+internal class AddressFieldElementRepository @Inject internal constructor(
```

Comment:
If you just add `@Singleton` to this, `ResourceRepositoryModule.provideAddressFieldRepository` can be deleted.

Generally when a constructor is marked `@Inject`, it tells dagger all its parameters are already provided somewhere and the constructor should not be called explicitly

```
@Singleton
internal class AddressFieldElementRepository @Inject internal constructor(
    val resources: Resources
)
```